docs/getting-started/create-configuration.md

```diff
@@ -591,7 +591,7 @@ you may specify a specific package by using the `-f` flag.
 ```console
 # Set this to the Docker Hub username or OCI registry you wish to use.
 REG=my-package-repo
-kubectl crossplane push configuration ${REG}/getting-started-with-aws:1.2.2
+kubectl crossplane push configuration ${REG}/getting-started-with-aws:v1.2.3
```

Comment:
These previously had the leading `v` stripped inadvertently, so it has been re-added.